tst/CTA.FeatureDetection.Tests/Fixtures/TestProjectsSetupFixture.cs

```diff
@@ -46,6 +48,7 @@ public class TestProjectsSetupFixture
         public static string CoreWebApiSolutionPath => Path.Combine(_targetDir, _coreWebApiDir, CoreWebApiSolutionName);
         public static string EfSolutionPath => Path.Combine(_targetDir, _efDir, EfSolutionName);
         public static string MvcSolutionPath => Path.Combine(_targetDir, _mvcDir, MvcSolutionName);
+        public static string WebFormsSolutionPath => @"D:\TestRepos\TestProjects\net472\FeatureDetection\WebForms\ASP.NET-WebForms.sln";
```

Comment:
Can we replace the string literal with `Path.Combine(...)` like other solution paths?

e.g. `WebFormsSolutionPath => Path.Combine(_targetDir, _webFormsDir, WebFormsSolutionName);`